catalyst/dl/scripts/run.py

```diff
@@ -105,18 +107,17 @@ def main_worker(args, unknown_args):
     utils.set_global_seed(args.seed)
     utils.prepare_cudnn(args.deterministic, args.benchmark)
 
-    config.setdefault("distributed_params", {})
+    config = OmegaConf.to_container(config, resolve=True)
```

Comment:
This brings me to the next question:
What happens if you use DictConfig as is?
for most purposes, it's a drop in replacement to a regular dict.

Reply:
I have already try it, but in some cases Dict can not be replaced by DictConfig (for example json.dump() method).